zerver/management/commands/send_realm_reactivation_email.py

```diff
@@ -0,0 +1,30 @@
+
+from argparse import ArgumentParser
+
+from zerver.lib.management import ZulipBaseCommand
+from zerver.lib.send_email import send_email, FromAddress
+from confirmation.models import create_confirmation_link, Confirmation
+
+from typing import Any
+
+class Command(ZulipBaseCommand):
+    help = """Emails realm admins with reactivation link"""
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        self.add_realm_args(parser, True)
+
+    def handle(self, *args: Any, **options: str) -> None:
+        realm = self.get_realm(options)
+        assert realm is not None
+        if not realm.deactivated:
+            print("The realm", realm.name, "is already active.")
+            exit(1)
+        confirmation_url = create_confirmation_link(realm, realm.host, Confirmation.REALM_REACTIVATION)
+        context = {'confirmation_url': confirmation_url}
+        admins = realm.get_admin_users()
+        print('Sending email to admins')
+        for admin in admins:
+            send_email('zerver/emails/realm_reactivation', to_email=admin.email,
+                       from_address=FromAddress.tokenized_no_reply_address(),
+                       from_name="Zulip Account Security", context=context)
+        print('Done!')
```

Comment:
It would be better to move most of the core logic of this to a function `check_send_reactivation_email`, since I can imagine in the future we might support triggering the reactivation emails from a place that isn't a management command, and it's a lot cleaner to do that migration if we just have a `do_send_reactivation_email` function that does everything but the print statements.